src/workerd/io/worker.c++

```diff
@@ -2217,30 +2235,36 @@ public:
     return ioHandler.messagePump();
   }
 
+  void handleDispatchProtocolMessage(
```

Comment:
Here too, it seems like this was refactored to avoid allocating `jsg::V8StackScope` on a coroutine frame -- but changing to `jsg::runInV8Stack` actually solves that problem on its own, so the surrounding refactoring seems unnecessary?